src/Servers/IIS/build/Build.Settings

```diff
@@ -4,8 +4,7 @@
      <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">$(MSBuildThisFileDirectory)..\</SolutionDir>
      <Configuration Condition="'$(Configuration)'==''">Debug</Configuration>
      <Platform Condition="'$(Platform)' == ''">Win32</Platform>
-     <PlatformToolsetVersion Condition=" '$(VisualStudioVersion)' == '17.0' ">v143</PlatformToolsetVersion>
-     <PlatformToolsetVersion Condition=" '$(PlatformToolsetVersion)' == '' ">v142</PlatformToolsetVersion>
+     <PlatformToolsetVersion>v142</PlatformToolsetVersion>
```

Comment:
Do we need this in both Directory.build.props and here?

Reply:
I still need to clean up some of the projects, the libs aren't including build.settings, only the dlls are, I'll try to normalize some of these in the next iteration